### ISSUE TITLE
False positive PhanUnreferencedUseNormal

### ISSUE THREAD

**Issue by a-menshchikov** [Drenso/PhanExtensions#12]
The following code is correct, but "Possibly zero references to use statement for classlike/namespace LdapGroupRoleRepository"

```
<?php

declare(strict_types=1);

namespace App\Entity\User;

use App\Repository\User\LdapGroupRoleRepository;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=LdapGroupRoleRepository::class)
 */
class LdapGroupRole
{
...
```

**Comment by bobvandevijver** [comment]
Hmm, that is an interesting scenario. That would require us to fully regex the annotation, which might be quite complex. Should we assume that a class is always assumed when the `::class` is found on a line, with a valid class like in front of it?

**Comment by a-menshchikov** [comment]
Yes, I think so.

For more details see commit https://github.com/symfony/maker-bundle/commit/89a66497338a7658c5ddebf69c8a85b5ce20a92f .

**Comment by bobvandevijver** [comment]
A fix for this has been published in [v3.1.0](https://github.com/Drenso/PhanExtensions/releases/tag/v3.1.0).

**Comment by bobvandevijver** [comment]
I'm now experiencing a similar scenario which also triggers a false positive:

```php
use App\Security\Roles;

/**
* @IsGranted(Roles::DASHBOARD)
*/
public function dashboard(){}
```

```
src/Controller/DefaultController.php:12 PhanUnreferencedUseNormal Possibly zero references to use statement for classlike/namespace Roles (\App\Security\Roles)
```

I will look into solving this as well.
